src/community/gwc-azure-blob/src/main/java/org/geoserver/gwc/web/blob/AzureBlobStorePanel.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">	
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:wicket="http://wicket.apache.org/" >
+<body>
+	<wicket:panel>
+		<fieldset>
+			<label for="container"><wicket:message key="container"/></label>
+			<input id="container" wicket:id="container" type="text" class="text" />
+				
+			<label for="accountName"><wicket:message key="accountName"/></label>
+			<input id="accountName" wicket:id="accountName" type="text" class="text" />
+				
+			<label for="accountKey"><wicket:message key="accountKey"/></label>
+			<input id="accountKey" wicket:id="accountKey" type="text" class="text" />
+						
+			<label for="prefix"><wicket:message key="prefix"/></label>
+			<input id="prefix" wicket:id="prefix" type="text" class="text" />
+			
+			<label for="serviceURL"><wicket:message key="serviceURL"/></label>
+			<input id="serviceURL" wicket:id="serviceURL" type="text" class="text" />
+			
+			<label for="maxConnections"><wicket:message key="maxConnections"/></label>
+			<input id="maxConnections" wicket:id="maxConnections" type="text" class="text" />
+						
+			<label for="useHTTPS"><wicket:message key="useHTTPS"/></label>
+			<input id="useHTTPS" wicket:id="useHTTPS" type="checkbox"/>
+	
+			<label for="proxyHost"><wicket:message key="proxyHost"/></label>
+			<input id="proxyHost" wicket:id="proxyHost" type="text" class="text" />
+			
+			<label for="proxyPort"><wicket:message key="proxyPort"/></label>
+			<input id="proxyPort" wicket:id="proxyPort" type="text" class="text" />
+				
+			<label for="proxyUsername"><wicket:message key="proxyUsername" /></label>
+			<input id="proxyUsername" wicket:id="proxyUsername" type="text" class="text" />
+				
+			<label for="proxyPassword"><wicket:message key="proxyPassword" /></label>
+			<input id="proxyPassword" wicket:id="proxyPassword" type="text" class="text" />
+		</fieldset>
+	</wicket:panel>
+</body>
+</html>
```

Comment:
Newline at end of file if you're going to make other changes.

Reply:
Done